application/features/authentication/authentication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Authentication
+  In order to identify a specific user
+  As an authorized user
+  I need to be able to authenticate a user based on certain credentials
+
+  Scenario: Create an authentication for a known user with a matching password
+    Given I am authorized
+      And I receive a username
+      And I receive a password
+    When I verify the password matches the associated password
+    Then I will respond positively
+
+  Scenario: Attempt to create an authentication for an unknown user
+    When I realize the username does not exist in my system
```

Comment:
You might want some number of `Given`s before this `When`... perhaps something like this:
```
Given I am authorized
  And I receive a username
  And I receive a password
  But that username does not exist in my system
When I attempt to verify the username and password
Then I will respond negatively
```
Thoughts?